src/app/core/parameters/stringValue.ts

```diff
@@ -0,0 +1,16 @@
+import Parameter from "../parameter";
+
+export default class StringValue extends Parameter {
+    static readonly typeString: string = "String";
+    static readonly description: string = "StringValue must be a String, such as 'foobar'";
+
+    constructor(value: string | String) {
```

Comment:
Whats the difference between both these string types ?

Reply:
From what I understand, 'String' is an object, while 'string' is a primitive. There were other parts of the code which accounted for both so I added it here.